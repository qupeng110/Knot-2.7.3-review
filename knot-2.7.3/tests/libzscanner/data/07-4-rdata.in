@ TXT \# 2 015C