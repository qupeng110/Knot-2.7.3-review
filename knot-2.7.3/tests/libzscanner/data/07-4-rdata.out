OWNER=00
CLASS=0001
RRTTL=00000000
RTYPE=0010
RDATA=015C
------
